source/features/shorten-links.tsx

```diff
@@ -10,16 +10,24 @@ function init(): void {
 	observe(`a[href]:not(.${linkifiedURLClass})`, {
 		constructor: HTMLAnchorElement,
 		add(link) {
-			// Don't shorten links in code or code suggestions (but shorten them in review comments)
-			// Code explanation: Exclude links if the closest element found is not `.comment-body`
-			// https://github.com/refined-github/refined-github/pull/4759#discussion_r702460890
-			if (link.closest('.blob-code, .comment-body, .js-suggested-changes-blob')?.matches('.blob-code, .js-suggested-changes-blob')) {
-				return;
+			if (!isLinkInCode(link)) {
+				applyToLink(link, location.href);
 			}
-
-			applyToLink(link, location.href);
 		},
 	});
+
+	for (const link of document.querySelectorAll('.comment-body a[href*="/tree/"]')) {
```

Comment:
This already happens because `observe` automatically calls `add` on load, on every `a[href]`, which includes these.